### ISSUE TITLE
Nested flux templates produce unexpected results

### ISSUE THREAD

**Issue by backbone87** [influxdata/influxdb-client-js#251]
Also switching to `fluxExpression` doesnt work either (for empty expressions) as it errors when used inside flux tagged template.

__Expected Behavior:__
Nesting flux templates should work and produce same results as if it where not nested

__Current Behavior:__
Nesting flux templates interpret the nested template as a string (which is to be escaped)

__Possible Solution:__
Cause by https://github.com/influxdata/influxdb-client-js/blob/master/packages/core/src/query/flux.ts#L225

__Steps/Code to Reproduce the Problem:__
```
const inner = flux`last()`;
const outer = flux`from(bucket: "my-bucket") |> ${inner} |> yield()`;
// outer.toString() is: `from(bucket: "my-bucket") |> "last()" |> yield()`;
```

__Specifications:__
- Version: 1.6
- Platform: node:12

edit: there is a test case for this, but the CI does not execute the test suite correctly and falsely passes

**Comment by sranka** [comment]
@backbone87 that you for reporting this, I am on it

**Comment by sranka** [comment]
additionally: the following test must pass as well
```
    const empty = flux``
    expect(flux`from(bucket:"my-bucket")${empty}`.toString()).equals(
      'from(bucket:"my-bucket")'
    )
```

**Comment by sranka** [comment]
@backbone87 the fix is available since 1.7.0-nightly.10, the faking tests were repaired in the last PRs
